client/runtime_test.go

```diff
@@ -188,8 +189,14 @@ func TestRuntime_TLSAuthConfigWithLoadedCAPoolAndLoadedCA(t *testing.T) {
 	cert, err := x509.ParseCertificate(block.Bytes)
 	require.NoError(t, err)
 
-	pool, err := x509.SystemCertPool()
-	require.NoError(t, err)
+	var pool *x509.CertPool
+	if goruntime.GOOS == "windows" {
+		// Windows doesn't have the system cert pool.
+		pool = x509.NewCertPool()
+	} else {
+		pool, err = x509.SystemCertPool()
```

Comment:
Does it make sense to not use the system cert pool at all?

Reply:
For this test I think this is fine.